docs: update comprehend doc

DenovVasil opened the pull request:
[issue](https://github.com/camunda/connectors/issues/3551)

**Description**
 I just removed redundant parts of the documentation and changed the name of the fields

**When should this change go live?**
This is part of a scheduled alpha or minor. (apply alpha or minor label)

**PR Checklist**
 My changes are for the next minor and are in /docs directory (aka /next/).

akeller commented:
@DenovVasil, could you please use our PR template? This helps us scope and prioritize reviews. Your linked issue is completely empty. Given that we have no context for this change, @christinaausley would only be able to provide a grammatical review.

Until this is updated, I will place the `hold` label and block this from merging.

DenovVasil commented:
> @DenovVasil, could you please use our PR template? This helps us scope and prioritize reviews. Your linked issue is completely empty. Given that we have no context for this change, @christinaausley would only be able to provide a grammatical review.
> 
> Until this is updated, I will place the `hold` label and block this from merging.

@akeller I updated description.
FYI I just removed redundant parts of the documentation and changed the names of the fields. So I hope the review will be easy for you 😊

DenovVasil commented:
@chillleader  Tested it, works great. Thank you very much, this update helped me a lot 🙂

christinaausley commented on docs/components/connectors/out-of-the-box-connectors/amazon-comprehend.md:
```diff
@@ -74,36 +74,32 @@ For more details on the fields that can be configured during asynchronous execut
 
-- **Volume Kms Key Id**: ID for the AWS Key Management Service (KMS) key that Amazon Comprehend uses to encrypt data on the storage volume attached to the ML compute instance(s) that process the analysis job.
-- **Security group ids**: The ID number for a security group on an instance of your private VPC. For more details, refer to [security group](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_VpcConfig.html#comprehend-Type-VpcConfig-SecurityGroupIds).
+- **VolumeKmsKeyId**: ID for the AWS Key Management Service (KMS) key that Amazon Comprehend uses to encrypt data on the storage volume attached to the ML compute instance(s) that process the analysis job.
```
These fields are exactly as they are referenced in the UI including casing, yes? If so, you have my approval 👍
